gatsby_fp/src/pages/404.js

```diff
@@ -1,14 +1,19 @@
 import React from "react"
-
-import Layout from "../components/Layout"
-import SEO from "../components/Seo"
+import Head from "../components/Layout/Head"
+import Header from "../components/Layout/Body/Header"
+import Body from "../components/Layout/Body"
 
 const NotFoundPage = () => (
-  <Layout>
-    <SEO title="Not Found - Stevens Institute of Technology" />
-    <h1>NOT FOUND</h1>
-    <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
-  </Layout>
+  <>
+    <Head
+      pageTitle={`Not Found - Stevens Institute of Technology`}
+    />
+    <Header/>
```

Comment:
This needs to be removed since the <Body> component already includes it. Otherwise we get a double header as seen at https://deploy-preview-17--stevens-fp.netlify.app/foo